static/src/javascripts/projects/commercial/modules/dfp/queue-advert.ts

```diff
@@ -0,0 +1,11 @@
+import type { Advert } from './Advert';
+import { dfpEnv } from './dfp-env';
+
+export const queueAdvert = (advert: Advert): void => {
+	dfpEnv.advertsToLoad.push(advert);
+	// Add to the array of ads to be refreshed (when the breakpoint changes)
+	// only if its `data-refresh` attribute isn't set to false.
+	if (advert.node.getAttribute('data-refresh') !== 'false') {
```

Comment:
Could we use `dataset`?

```suggestion
	if (advert.node.dataset.refresh !== 'false') {
```